src/kokkos/KokkosCore/AtomicPairCounter.h

```diff
@@ -0,0 +1,46 @@
+#ifndef HeterogeneousCore_CUDAUtilities_interface_AtomicPairCounter_h
+#define HeterogeneousCore_CUDAUtilities_interface_AtomicPairCounter_h
+
+#include <cstdint>
+#include "KokkosCore/InitializeScopeGuard.h"
```

Comment:
This `#include` should go in the unit test source file, and the `#include "KokkosCore/kokkosConfigCommon.h"` can be removed there.